packaging/nmstate.py3.spec

```diff
@@ -36,6 +37,7 @@ This package contains the Python 3 library for Nmstate.
 
 %prep
 %setup -q
+sed -i -e '/dbus/d' requirements.txt
```

Comment:
IMHO it would be better to match the full requirement, i.e. `^dbus-python`.

Reply:
Fixed. Now using `sed -i -e '/^dbus-python$/d' requirements.txt`